### DISCUSSION TITLE
Option to set used keys on exported replays for stable view

### DISCUSSION THREAD

**Discussion by tuszmak** [ppy/osu#29926] [Ideas] [answered]
Currently when you export a lazer replay and view it in stable, the lazer keybinds 1 and 2 are the two mouse buttons. 
Since lazer doesn't differentiate between mouse and keyboard input, would it be possible to add an option to set how stable interprets these inputs? (like lazer keybind 1 is stable keyboard input 1, keybind 2 is keyboard 2 and keyboard 3 is mouse 1)

Its place could be a popup when you export a replay, or a ctrl+o menu option

**Comment by bdach** [comment, marked as answer]
It's not possible without splitting the two keybinds into four. Which is not gonna happen.

**Reply by peppy** [reply]
I'm not sure I'd say never. A lot of people have asked for this to return 😅.
